snare.py

```diff
@@ -463,15 +462,16 @@ def parse_timeout(timeout):
         print("--page-dir: {0} does not exist".format(args.page_dir))
         exit()
     args.index_page = os.path.join("/", args.index_page)
-    
+
     if not os.path.exists(os.path.join(full_page_path, 'meta.json')):
         conv = Converter()
         conv.convert(full_page_path)
         print("pages was converted. Try to clone again for the better result.")
 
     with open(os.path.join(full_page_path, 'meta.json')) as meta:
         meta_info = json.load(meta)
-    if not os.path.exists(os.path.join(base_page_path,args.page_dir,os.path.join(meta_info[args.index_page]['hash']))):
+    if not os.path.exists
```

Comment:
this line is not working because of wrapping
```
  File "snare.py", line 473
    if not os.path.exists
                        ^
SyntaxError: invalid syntax

```

Reply:
@afeena  I have resolved the issue. Please take a look

Reply:
@afeena I have fixed the issue. Please take a look.

Reply:
@mzfr Are you sure you commit your changes? It seems like the same problem still occurs